imports/client/components/App.tsx

```diff
@@ -87,7 +93,7 @@ class AppNavbar extends React.Component<AppNavbarProps> {
   }
 }
 
-const AppNavbarContainer = withTracker(() => {
+const AppNavbarContainer = withTracker((_props: RouteComponentProps) => {
```

Comment:
Because we do make use of it in `AppNavbar` (so that we can access the history object on logout, so that we redirect before other components have a chance to observe `Meteor.userId()` returning null).

And if I don't annotate the type here, then typescript complains (because the props passed to `AppNavbar` might not include all of the `RouteComponentProps`, which we need them to).